src/RestServer/Controllers/v1/WalletController.cs

```diff
@@ -0,0 +1,646 @@
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpPost("{session:required}/import", Name = "WalletImportByWif")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(WalletAddressModel))]
+        public IActionResult WalletImportPrivateKey(
+            [FromRoute(Name = "session")]
+            Guid sessionId,
+            [FromBody]
+            WalletImportKey model)
+        {
+            if (WalletSessions.ContainsKey(sessionId) == false)
+                throw new KeyNotFoundException(sessionId.ToString("n"));
+            var session = WalletSessions[sessionId];
+            session.ResetExpiration();
+            var wallet = session.Wallet;
+            var account = wallet.Import(model.Wif);
+            if (account == null)
+                throw new WalletException("Account couldn\'t be imported.");
+            if (wallet is NEP6Wallet nep6)
+                nep6.Save();
+            return Ok(new WalletAddressModel()
+            {
+                Address = account.Address,
+                ScriptHash = account.ScriptHash,
+                Publickey = account.GetKey().PublicKey,
+                HasKey = account.HasKey,
+                Label = account.Label,
+                WatchOnly = account.WatchOnly,
+            });
+        }
+
+        /// <summary>
+        /// List all the addresses in the wallet.
+        /// </summary>
+        /// <param name="sessionId" example="066843daf5ce45aba803587780998cdb">Session Id of the open/created wallet.</param>
+        /// <returns>An array of wallet address objects.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("{session:required}/address/list", Name = "GetWalletListAddress")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(WalletAddressModel[]))]
+        public IActionResult WalletListAddresses(
+            [FromRoute(Name = "session")]
+            Guid sessionId)
+        {
+            if (WalletSessions.ContainsKey(sessionId) == false)
+                throw new KeyNotFoundException(sessionId.ToString("n"));
+            var session = WalletSessions[sessionId];
+            session.ResetExpiration();
+            var wallet = session.Wallet;
+            var accounts = new List<WalletAddressModel>();
+            foreach (var account in wallet.GetAccounts())
+                accounts.Add(new WalletAddressModel()
+                {
+                    Address = account.Address,
+                    ScriptHash = account.ScriptHash,
+                    Publickey = account.GetKey().PublicKey,
+                    HasKey = account.HasKey,
+                    Label = account.Label,
+                    WatchOnly = account.WatchOnly,
+                });
```

Comment:
```suggestion
            using var snapshot = _neosystem.GetSnapshot();
            foreach (var account in wallet.GetAccounts())
            {
                var contract = account.Contract;
                var type = "Nonstandard";

                if (account.WatchOnly)
                {
                    type = "WatchOnly";
                }
                else if (IsMultiSigContract(contract.Script))
                {
                    type = "MultiSignature";
                }
                else if (IsSignatureContract(contract.Script))
                {
                    type = "Standard";
                }
                else if (NativeContract.ContractManagement.GetContract(snapshot, account.ScriptHash) != null)
                {
                    type = "Deployed-Nonstandard";
                }
                accounts.Add(new WalletAddressModel()
                {
                    Address = account.Address,
                    ScriptHash = account.ScriptHash,
                    Publickey = account.GetKey()?.PublicKey,
                    HasKey = account.HasKey,
                    Type = type,
                    Label = account.Label,
                    WatchOnly = account.WatchOnly,
                });
            }
```
Add  address `type`, return null when publickey can't be got when `WatchOnly` or `Multisig`.

Reply:
You didn't Add `Type` to the Model?